lib/tunnel.js

```diff
@@ -732,6 +732,51 @@ function addArguments_(self, fnCallback) {
             binaryArgs.push(value); // will work only if mode === "ssh", allowed values (over_22, over_443, over_ws), can give error if any other value specified
           }
           break;
+
+        case 'config':
+          if (value) {
+            binaryArgs.push('--config');
+            binaryArgs.push(value);
+          }
+          break;
+
+        case 'clientcert':
+          if (value) {
+            binaryArgs.push('--clientCert');
+            binaryArgs.push(value);
+          }
+          break;
+
+        case 'clientkey':
+          if (value) {
+            binaryArgs.push('--clientKey');
+            binaryArgs.push(value);
+          }
+          break;
+        
+
+        case 'mtlshosts':
+          if (value) {
+          binaryArgs.push('--mTLSHosts');
+          binaryArgs.push(value);
+        }
+          break;
+
+
+        case 'pacfile':
+          if (value) {
+          binaryArgs.push('--pacfile');
+          binaryArgs.push(value);
+        }
+          break;
+        
+        case 'allowhosts':
+          if (value) {
+          binaryArgs.push('--allowHosts');
```

Comment:
FYI you can just do `foo.push('--opt', value)` and cuts the `push` calls in half everywhere in this file :)